src/app/shared/utils/phone-helper.ts

```diff
@@ -0,0 +1,23 @@
+import { AccommodationOffer } from '@app/core/api';
+import { PREFIXES } from '../consts';
+import { PhoneNumber } from '../models/phone.model';
+
+export const PHONE_HELPER = {
+  initPhoneOnEdit: (context: any) => {
+    context.accommodationsResourceService.getAccommodations(context.offerId).subscribe((resp: AccommodationOffer) => {
+      context.phone.phoneNumber = resp.phoneNumber || '';
+      if (resp.phoneCountryCode) {
+        context.phone.prefix = PREFIXES.find((v) => v.prefix === resp.phoneCountryCode)?.prefix || '';
+      }
+      context.data = resp;
+    });
+  },
+
+  preparePhoneNumber: (context: any) => {
+    if (context.phone.phoneNumber) {
+      context.data.phoneNumber = `${context.phone.prefix}${context.phone.phoneNumber}`;
+    } else {
+      context.data.phoneNumber = undefined;
+    }
```

Comment:
`context.data.phoneNumber = context.phone.phoneNumber ? 
  context.phone.prefix.concat(context.phone.phoneNumber) : undefined`

Reply:
Musicie się zdecyfować, ostatnio zmieniałem z ternary na normale ify elsy bo za mało czytelne dla Was było :)

Reply:
Jedno ternary jest czytelne. Ternary w ternary już gorzej :) każdy przypadek rozpatrujemy oddzielnie

Reply:
Z tym że chodziło dokładnie o ten przypadek, @mgol nalegał żeby zmienić, więc zostawiam to jak jest, bo mogę tak to zmieniać co 2 tygodnie ;)